### ISSUE TITLE
Need support for ARM platforms (native .so library for android)

### ISSUE THREAD

**Issue by longnetwork** [iceland2k14/secp256k1#7]
For use on microconsumption devices

**Comment by iceland2k14** [comment]
Yes i want to add this eventually but not decided yet as i was thinking to add multicpu code in the some array functions. Let me think how i am going to utilize it. Perhaps few main functions in android if not all.

**Comment by arashisec** [comment]
waiting support ARM device.

**Comment by AhmadShamli** [comment]
also waiting for ARM cpu support..not necessarily mean for phone..but also for some server with arm cpu...

**Comment by shapeshifte** [comment]
So, on android it requires the .so file to be 32 bits instead of 64 bits? I'm trying it on pydroid3, is there any news about android support?
